Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -112,8 +112,8 @@
         <item name="android:textColor">@color/button_text_color_emphasized</item>
     </style>
 
-    <style name="debugLogButton" parent="button">
-        <item name="android:backgroundTint">@color/button_debug_log</item>
+    <style name="buttonBarAlertRed" parent="button">
```

Comment:
I'm a little late to the party, but don't we already have a red button? Settings > Reset